src/utils/random.js

```diff
@@ -3,4 +3,8 @@ function getRandomElement(array) {
   return array[n];
 }
 
-export { getRandomElement };
+function fullName({ firstname, lastname }) {
```

Comment:
IMHO this util function should be placed into another file called `users.js`.
Initially, I created `random.js` to gather some random generation/access methods.
Another solution would be to keep the code as is and rename this file to be `misc.js`.